### PR TITLE
Add new test deployment mechanism to moped

### PR DESCRIPTION
**Pull request by frankhereford**
The very definition of WIP.

edited by Chia 1/30/23

We've discussed a bit in meetings to go ahead and merge this into main, knowing that it could definitely be refined and refactored. However, this is what is currently being used to deploy test moped instances (has been in production for some time). 

Companion atd-prefect PR: https://github.com/cityofaustin/atd-prefect/pull/43

I removed the SQS functions from the github action, since those are not being used.

### CODE REVIEW

File: moped-editor/netlify.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #   From: https://docs.netlify.com/site-deploys/overview/#deploy-contexts
 #   Also: https://docs.netlify.com/configure-builds/environment-variables/
 [context.branch-deploy]
-    command = "echo 'Build Context Mode: Branch-Deploy' && npm run build:netlifypr"
+    command = "echo 'Build Context Mode: Branch-Deploy' && npm run build:netlifybranch"
```
Comment (frankhereford) [inline moped-editor/netlify.toml:8]: This is going to make this a bit of a leap of faith. Netlify deploys will not respect the new system until this is in the branches that are being deployed.

**Review by chiaberry — COMMENTED** [review]
check to see if the aws-moped-sqs-helper.sh is being used

**Review by chiaberry — APPROVED** [review]

**Review by mddilley — APPROVED** [review]
YOLO!!! 🚀 😄 But, for real, I think this code has more than proven itself for our Moped test deployments. I did forget that we updated the path to the Airflow ETL code in here. Does this have any implications for our ETLs? Not sure if the Airflow config needs to point to the new directory.

**Review by roseeichelmann — APPROVED** [review]

**Comment by frankhereford** [comment]
I'm going to close this; it's been almost two years and it's 4K commits behind `main`. I think we're good! 😄

**Pull request closed without merging**